### DISCUSSION TITLE
Editing spreadsheet

### DISCUSSION THREAD

**Discussion by semibreave** [dfinke/ImportExcel#1554] [General]
I want to know if this module is all about importing spreadsheet into an object or possible to edit the existing spreadsheet(no importing) as well? Thank you very much

**Comment by dfinke** [comment]
You can use `Open-ExcelPackage` and `Close-ExcelPackage` to open the xlsx and then access sheets, cells etc to edit existing sheets.

**Reply by semibreave** [reply]
Thank you. Just what I needed Doug
